api/src/main/resources/workbench-api.yaml

```diff
@@ -4199,7 +4199,6 @@ definitions:
         type: string
       approved:
         type: boolean
-        default:
```

Comment:
These were the cause of the test breakages - it was resulting in a default value of `false`. I suspect the Swagger plugin may convert to JSON before passing to the codegen CLI, which would explain why the resulting code was different. I didn't see a good reason to keep these lines, though perhaps I could have also done `default: null`, if you want to leave them around.

Reply:
tl;dr, if you really want 3-state boolean semantics (🤮 ), then you need `default: null`.

I put in a PR a while back to default all booleans explicitly because the null default made them all tri-state, and since we don't  have proper constructors, we kept having  issues. There are only a couple of places where we want null booleans. (I'm planning on writing about this in one of the Java best practices guides: IMO if you need null booleans, you need an enum). It's also a painful mapping to typescript.

So the new convention I proposed was to put a default value for all booleans in Swagger files, and this new issue seems to me to be an even stronger argument  for it.

Reply:
I tried `default: null` as well. It doesn't work with the plugin, it has the same behavior as `default: `, which both appear to be interpreted as default: false

Reply:
That's definitely a regression then. That, or we were relying on undocumented behavior, which I don't believe we were.

Reply:
What is the regression? Having to omit the default rather than stating `default: null` would be a change, but it seems pretty insignificant and non-functional to me. If you're concerned about documentation, I can replace these instances with a comment.

Reply:
I mean a boolean with no stated default changing causing generated classes to initialize members to `false` instead of `null` seems like a potentially (silently) breaking change.

Reply:
I'm not too surprised to see undefined behavior out of `default: null` and especially `default: `. I'm probably not going to spend more time investigating this issue unless you have clear documentation that Java code gen in supposed to respond to `default: null` in a specific way. Even then, it's probably a idiosyncrasy with how the plugin works - best I can do is file a bug, this issue is not severe enough to back out of this approach.

To be clear, this has nothing to do with Swagger 3. Everything is still on Swagger 2. The change here is to use a plugin instead of our own bespoke codegen task.